test/integration/multicluster/multicluster-traffic/federated_test.go

```diff
@@ -0,0 +1,151 @@
+package multiclustertraffic
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/linkerd/linkerd2/testutil"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+// TestFederatedService deploys emojivoto to two clusters and has the web-svc
+// in both clusters join a federated service. It creates a vote-bot in the
+// source cluster which sends traffic to the federated service and then checks
+// the logs of the web-svc in both clusters. If it has successfully issued
+// requests, then we'll see log messages indicating that the web-svc can't
+// reach the voting-svc (because it's not running).
```

Comment:
not that voting-svc isn't running, but that we're using the failed :doughnut: requests to attest web-svc is getting called.